adapters/info_test.go

```diff
@@ -56,6 +58,7 @@ func TestSiteNotSupported(t *testing.T) {
 }
 
 func TestImpFiltering(t *testing.T) {
+	// Set up
```

Comment:
Nitpick: The comments don't seem helpful here. Please consider removing.

Reply:
removed